kubemarine/thirdparties.py

```diff
@@ -200,18 +200,24 @@ def install_thirdparty(filter_group: NodeGroup, destination: str) -> NodeGroupRe
         extension = destination.split('.')[-1]
         if extension == 'zip':
             cluster.log.verbose('Unzip will be used for unpacking')
-            # TODO re-installation waits forever
-            remote_commands += ' && sudo unzip %s -d %s' % (destination, config['unpack'])
+            remote_commands += ' && sudo unzip -o %s -d %s' % (destination, config['unpack'])
+            
+            remote_commands += ' && sudo chmod  %s %s' \
```

Comment:
`config['unpack']` is a directory where this thirdparty should be unpacked, so `chmod` and `chown` commands should be applied to `config['unpack] + destination`, not just to `config['unpack']`.

Reply:
As per discussion, chnages have been made to chnage the ownership and file permissions for the content inside unpacked directory instead of `config['unpack']`